networkpolicy/README.md

```diff
@@ -6,18 +6,25 @@ Tools to configure OpenShift with a set of baseline [NetworkPolicy](https://docs
 
 The [OpenShift SDN](https://docs.openshift.com/container-platform/latest/install_config/configuring_sdn.html) offers several plugin implementations. One of these plugins supports NetworkPolicy, which provides for more fine grained policies for communication between pods running on the cluster. Once enabled, all communication between pods within OpenShift are not restricted in any way. The tools within this repository will configure an OpenShift environment with a set of default policies that support a [zero trust network](https://tigera.io/wp-content/uploads/2017/12/wp-tigera-zero-trust-cloud-native-environment.pdf).
 
-The following sections describes the relevant concepts along with tooling to automate the application of these policies to an OpenShift environment. 
+The following sections describes the relevant concepts along with tooling to automate the application of these policies to an OpenShift environment.
 
 ## Concepts
 
 ### Deny Traffic By Default
 
-Since NetworkPolicy does not restrict pod communication by default, this would violate many of the benefits that are enforced by other SDN plugins (such as the _ovs-multitenant_ SDN plugin. To support a zero trust network policy, all components within the cluster should not be able to communicate with each other unless a Network Policy is configured. This requires the application of a policy to deny traffic within each project. A [default-deny.yml](baseline/default-deny.yml) NetworkPolicy is available which can be applied to each project to enforce this rule.
+Since NetworkPolicy does not restrict pod communication by default, this would violate many of the benefits that are enforced by other SDN plugins (such as the _ovs-multitenant_ SDN plugin). To support a zero trust network policy, all components within the cluster should not be able to communicate with each other unless a Network Policy is configured. This requires the application of a policy to deny traffic within each project. A [default-deny.yml](policies/baseline/default-deny.yml) NetworkPolicy is available which can be applied to each project to enforce this rule.
 
+### Deny Egress Traffic by Default
+
+OpenShift provides a similar `EgressNetworkPolicy` object for controlling outbound traffic leaving the cluster. A [deny-external-egress.yml](policies/baseline/deny-external-egress.yml) EgressNetworkPolicy is available to be defined which will drop all connections attempting to leave the cluster from the project in which it is defined.
```

Comment:
The referenced `EgressNetworkPolicy` does allow access to github.com. Maybe this should be mentioned in order to avoid confusion that this policy does in fact deny all traffic